user_docs/en/changes.t2t

```diff
@@ -36,6 +36,17 @@ What's New in NVDA
 - A landmark or region's name is always automatically spoken when jumping inside from outside using quick navigation or focus in browse mode. (#13307)
 - When beep or announce 'cap' for capitals is enabled with delayed character descriptions, NVDA no longer beeps or announces 'cap' twice. (#14239)
 - Controls in tables in Java applications will now be announced more accurately by NVDA. (#14347)
+- The following settings will no longer be unexpectedly different when used with multiple profiles. (#14170)
+  - Line indentation in Document formatting settings.
+  - Cell borders in doc formatting settings
+  - Show messages in braille settings
+  - Tether Braille in braille settings
+  - In some rare cases, the following settings used in profiles may be unexpectedly modified when installing this version of NVDA:

```

Comment:
@feerrenrut 
Actually, it seems that I repeat the name of the options two times. Maybe I could just write as below and remove the second list of options?

```suggestion
  - In some rare cases, these settings used in profiles may be unexpectedly modified when installing this version of NVDA.
```

Reply:
Ah, ok. I'll adjust this, how about this:
```
- Some settings will no longer be unexpectedly different when used with multiple profiles. (#14170)
  - The following settings have been addressed:
    - Line indentation in Document formatting settings.
    - Cell borders in doc formatting settings
    - Show messages in braille settings
    - Tether Braille in braille settings
  - In some rare cases, these settings used in profiles may be unexpectedly modified when installing this version of NVDA.
  - Please check these options in your profiles after upgrading NVDA to this version.
  -
```

Reply:
That's OK. Thanks.